src/Util.cpp

```diff
@@ -1,8 +1,22 @@
 #include <stdexcept>
+#include <algorithm>
+#include <iostream>
 
+#include "ale.h"
```

Comment:
This seems like it could be removed but I'm not sure. Does this use anything from ale.h? Or does it need it for the ale namespace?